docs/software/programming.md

```diff
@@ -66,25 +80,56 @@ Go is used by some of the big organizations such as Google, BBC, Uber, Soundclou
 
     To edit the code you can use an editor of your choice, or use [VSCodium](#vscodium). The _Go extension_ available in VSCodium provides rich language support for the Go programming language.
 
-    Just to have a taste of it, you could run some commands in this [_Go playground_](https://play.golang.org/p/AAX1cLCmA1c)
+    Just to have a taste of it, you could run some commands in [_Go playground_](https://play.golang.org/p/AAX1cLCmA1c)
 
     Checkout also the official tutorial [Get started with Go](https://golang.org/doc/tutorial/getting-started).
 
-=== "Install / Uninstall"
+=== "Install / Uninstall / Update"
 
     - We switched to automatic Go version detection. Here is an example:
 
-    ```sh
-    root@DietPi3:~# go version
-    go version go1.16.3 linux/arm
-    ```
+        ```sh
+        root@DietPi3:~# go version
+        go version go1.16.3 linux/arm
+        ```
+
+    - When the Go package gets uninstalled, the folder `/mnt/dietpi_userdata/go` is still kept.  
+      This is the place where packages are installed, custom compilations are run, sources are downloaded etc. It is especially important to keep it as long as we don't have a good dependency system that blocks uninstalls of dependencies. Otherwise it would be possible to uninstall Go while [OpenBazaar](../social/#openbazaar) is still installed. As a side effect, removing `/mnt/dietpi_userdata/go` would mean also removing [OpenBazaar](../social/#openbazaar).
+
+    - To update the Go package, execute the following:
```

Comment:
Instead of doing all those steps manually:
```
dietpi-software reinstall 188
```

Shall we provide this via dedicated "Update to latest version" tab to align with other install options?